tests/test_ephys_data_set.py

```diff
@@ -49,22 +49,22 @@ def test_emtpy_table_has_header():
     assert list(ds.filtered_sweep_table(sweep_number=7)) == list(ds.sweep_table)
 
 
-def test_get_sweep_number_by_stimulus_name_invalid_sweep():
+def test_get_sweep_number_invalid_sweep():
 
     with pytest.raises(IndexError):
         ds = get_dataset()
-        ds.get_sweep_number_by_stimulus_names(['I_DONT_EXIST'])
+        ds.get_sweep_number(['I_DONT_EXIST'])
```

Comment:
Don't you need some tests for `get_sweep_number` where `clamp_mode` is not the default value of `None`?

Reply:
Yes a test would be good.